addons/binding/org.openhab.binding.edimax/src/main/java/org/openhab/binding/edimax/handler/EdimaxHandler.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.edimax.handler;
+
+import java.io.IOException;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.eclipse.smarthome.core.types.RefreshType;
+import org.openhab.binding.edimax.EdimaxBindingConstants;
+import org.openhab.binding.edimax.configuration.EdimaxConfiguration;
+import org.openhab.binding.edimax.internal.ConnectionInformation;
+import org.openhab.binding.edimax.internal.commands.GetState;
+import org.openhab.binding.edimax.internal.commands.SetState;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link EdimaxHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Falk Harnisch - Initial contribution
+ */
+public class EdimaxHandler extends BaseThingHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EdimaxHandler.class);
+
+    private static final int PORT = 10000;
+
+    /**
+     * Connection Information to the device.
+     */
+    protected ConnectionInformation ci;
+
+    private ScheduledFuture<?> pollingJob;
+
+    /**
+     * Constructor for the edimax things.
+     *
+     * @param thing The thing
+     */
+    public EdimaxHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        LOGGER.debug("command for {}: {}", channelUID.getAsString(), command.toString());
```

Comment:
You don't have to call toString it will be called automatically, but only when the logging-level actually applies. Please check all.